### DISCUSSION TITLE
Multinode Treepicker should display already selected nodes

### DISCUSSION THREAD

**Discussion by johancoens** [umbraco/Umbraco-CMS#12829] [Features and ideas]
When using the multinode treepicker, the end user is able to select nodes in a tree. When there are already items selected, the tree does not show which nodes are already selected. Especially when using larger node trees this becomes rather unfriendly for the end user.

Suggested feature is to display the already selected nodes, and eventually also be able to deselect nodes from the tree picker.
![Multinode-treepicker](https://user-images.githubusercontent.com/765710/184318925-2364a801-f052-47a6-9fcf-094e5ce4caf7.gif)

**Comment by fransmilou** [comment]
+1 This is a very annoying limitation today.

**Comment by mrflo** [comment]
+1 I agree it should be greyed out
For new we are just checking if the id exists and if it's the case we don't do anything which is confusing
`if ($scope.ids.indexOf(itemId) < 0) {`
